app/main.py

```diff
@@ -1,8 +1,33 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power, clean_mark, average_rating, count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.clean_mark = clean_mark
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list):
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car):
+        return round(car.comfort_class * (self.clean_power - car.clean_mark) * self.count_of_ratings / self.distance_from_city_center, 1)
+
+    def wash_single_car(self, car):
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
```

Comment:
The `wash_single_car` method updates the `clean_mark` of a car but does not return any value or update the income. Consider returning the washing price from this method and updating the income in the `serve_cars` method.